go/riscv64.patch

```diff
@@ -28,3 +36,23 @@ index cece4cc..d368d3c 100644
  
    cp -a bin pkg src lib misc api test "$pkgdir/usr/lib/go"
    # We can't strip all binaries and libraries,
+diff --git cacheline.patch cacheline.patch
```

Comment:
Generally we store additional sources (like code patches) directly in the repo instead of creating them by `riscv64.patch`

Reply:
You mean it will be copied to the source dir automatically.